Jenkinsfile

```diff
@@ -76,7 +76,7 @@ pipeline {
                             buildEnvironment.overrideJDK(maven.javaVersion())
                         }
 
-                        sh "mvn -U -B -s $MAVEN_SETTINGS -Dfile.encoding=UTF-8 -DinstallAtEnd=true -DdeployAtEnd=true -Dsha1= -Dchangelist= -Drevision=$version clean install"
```

Comment:
Hvis du gjør dette vil du vel kunne ende opp med et korrupt lokalt m2-repository? Jeg klarte ikke å se at vi har konfigurert opp jobbene våre til å kjøre med repository isolation.

Case:
* Endring i to moduler (A & B)
** Signaturendring på metode i A
** Oppdatering i B for å bruke ny signatur i B
* Bygget feiler etter at A er installert
* Applikasjon X (som bruker A og B) bygger på samme node og får nå en A og B som ikke er kompatible